### ISSUE TITLE
Port forwarding, revisited

### ISSUE THREAD

**Issue by raubvogel** [boot2docker/boot2docker#689]
Correct me if I am wrong, but the port forwarding example shown in https://github.com/boot2docker/boot2docker/blob/master/doc/WORKAROUNDS.md assumes the vm client that is running boot2docker is in NAT mode, i.e. does not have its own IP in the network the vm host is at. But, if I have it in bridge mode so it now does have an IP in that network, shouldn't such workaround not be needed anymore?

**Comment by SvenDowideit** [comment]
that is correct, if you set up your vm differently from how boot2docker-cli does, then the instructions will apply less :)

I agree though, the boot2docker docs do assume you're using the boot2docker-cli

**Comment by dsebastien** [comment]
@SvenDowideit are there plans or ideas to improve/automate port forwarding w/ Boot2Docker?
